### DISCUSSION TITLE
Trying to pass custom configuration to core_letsencrypt addon service

### DISCUSSION THREAD

**Discussion by RanTheLab** [custom-components/pyscript#288] [Q&A]
I'm trying to call `core_letsencrypt` in a pyscript either by:
* `service.call('hassio', 'addon_start', addon='core_letsencrypt', ...)`
* `hassio.addon_start(addon='core_letsencrypt', ...)`

and would like to pass dynamic configuration parameters which are set in the Let's Encrypt addon configuration tab:

![image](https://user-images.githubusercontent.com/7691169/147615721-62a892e5-2a25-4bc8-b1b1-98e6f00bb222.png)

The `service.call()` works using what is configured on that tab, but I want to set a domain dynamically by passing the domain into my pyscript.  It is not clear to me how to do this reading [this doc](https://hacs-pyscript.readthedocs.io/en/stable/reference.html#calling-services).

**Comment by qui3xote** [comment]
**kwargs means that you can pass a dictionary containing any additional arguments you want to the service, and pyscript will pass them on to the service. In theory that means you could add {'domain':'my domain.com'}.

The real question is whether the letsencrypt service supports that. You might be able to tell by looking in the developer tools/services section of your home assistant interface.
